### ISSUE TITLE
Client currency rates: Cannot read property 'rate' of undefined

### ISSUE THREAD

**Issue by aaronjudd** [reactioncommerce/reaction#600]
Reported by @boboci9 and #258 and related to #366 

>Hi,
I found an issue in the main.js and in the line 23 I saw a comment // fix for https://github.com/reactioncommerce/reaction/issues/248 so I figured I should report it here. 
Everything is working fine and then suddenly after some form submits I start getting this error message in the console:
```
TypeError: Cannot read property 'rate' of undefined
    at main.js:30
```
>If it's any help I debugged the Locale variable before the if and when the error occurs the Locale object is filled like the Locale.locale is holding all info like { capital, continent, currency, etc} but the Locale.currency is an empty object.

**Comment by aaronjudd** [comment]
@zenweasel this sounds like it's right in the middle of the stuff you are debugging on  #366

**Comment by zenweasel** [comment]
Fix by 0bf9deedbee9601cc63ddf521e1dd1462b4962d5 by @newsiberian
